{"name": "Ошибка при неверном пароле", "status": "passed", "start": 1751895414357, "stop": 1751895415078, "uuid": "8d190c3e-6955-4e15-958e-ed706f050081", "historyId": "0036adbe4717b453c70c5db926a90f89", "testCaseId": "0036adbe4717b453c70c5db926a90f89", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_wrong_password", "labels": [{"name": "feature", "value": "Логин курьера"}, {"name": "epic", "value": "Курьеры"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-3L4TMJM"}, {"name": "thread", "value": "25140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}